Avatar/avatar.css

```diff
@@ -0,0 +1,220 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap");
+@import url("https://cdn.jsdelivr.net/npm/bootstrap-icons@1.3.0/font/bootstrap-icons.css");
+:root {
+    --vodka-UI-background-color: white;
```

Comment:
If these are colors common to the component library, you can add them to a separate shared file called `variables.css`. If not, then this looks good!